build.gradle

```diff
@@ -49,7 +49,9 @@ subprojects {
 
     dependencies {
       api platform("com.mx.path-core:platform:${project.ext.pathSDKVersion}")
+      implementation "com.mx.path-core:http"
```

Comment:
I think these dependencies should be in `realtime/build.gradle`

Reply:
fixed